tests/integration/native-image/mp-1/pom.xml

```diff
@@ -100,26 +55,20 @@
             </exclusions>
         </dependency>
         <dependency>
-            <groupId>io.helidon.common</groupId>
-            <artifactId>helidon-common</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>io.helidon.security.providers</groupId>
-            <artifactId>helidon-security-providers-http-auth</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>io.helidon.security.providers</groupId>
-            <artifactId>helidon-security-providers-oidc</artifactId>
+            <groupId>io.helidon.tracing</groupId>
+            <artifactId>helidon-tracing-jaeger</artifactId>
+            <scope>runtime</scope>
         </dependency>
         <dependency>
-            <groupId>org.glassfish.jersey.media</groupId>
-            <artifactId>jersey-media-json-binding</artifactId>
+            <groupId>io.helidon.microprofile</groupId>
+            <artifactId>helidon-microprofile-access-log</artifactId>
             <scope>runtime</scope>
         </dependency>
         <dependency>
             <groupId>org.jboss</groupId>
             <artifactId>jandex</artifactId>
             <version>2.1.1.Final</version>
```

Comment:
I know this was not changed as part of this PR but jandex is in `<dependencyManagement>` in `dependencies/pom.xml` so the version should not be needed here.

Reply:
Will fix in other tests as well.

Reply:
Fixed.